clients/client-shell/cmds/group/actors.go

```diff
@@ -36,6 +36,16 @@ func init() {
 
 	Command.AddCommand(statusCmd)
 
+	sealCmd := &cobra.Command{
+		Use:   "seal <taskGroupId>",
+		Short: "Seal the task group to disallow addition of new tasks",
+		Long:  "This operation is irreversible and calling it multiple times will only seal it once.",
+		RunE:  executeHelperE(runSeal),
+	}
+	sealCmd.Flags().BoolP("force", "f", false, "Skip sealing confirmation.")
+
+	Command.AddCommand(sealCmd)
+
```

Comment:
The new API method will already be available via `taskcluster api queue sealTaskGroup <taskGroupId>`, so I don't think this is needed.

Reply:
true! Thanks